pkg/kbuffer/handler/handler_test.go

```diff
@@ -227,14 +227,14 @@ func TestActivationHandler(t *testing.T) {
 					return resp, err
 				}
 				if e.attempts != "" {
-					resp.Header.Add(activator.RequestCountHTTPHeader, e.attempts)
+					resp.Header.Add(kbuffer.RequestCountHTTPHeader, e.attempts)
 				}
 				return resp, err
 			})
 
 			reporter := &fakeReporter{}
 			handler := ActivationHandler{
-				Activator: act,
+				KBuffer:   act,
```

Comment:
We should rethink the type structure here. I tend to think we should leave it as is (Activator) cause it's part of the ActivationHandler. Having KBuffer as part of the ActivationHandler feels weird.